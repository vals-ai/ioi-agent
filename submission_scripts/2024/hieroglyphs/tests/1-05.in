version https://git-lfs.github.com/spec/v1
oid sha256:60c595d6db1de3675af2e19c22bd83d3f26e6205dbb4b11fcdf0cbe39696df29
size 1177827
