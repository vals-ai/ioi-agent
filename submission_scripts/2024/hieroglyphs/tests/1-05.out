version https://git-lfs.github.com/spec/v1
oid sha256:0503d6322f592068a8d76cded8badc2d471c89e64b1dfa74c2dcd2f8a429d582
size 588933
